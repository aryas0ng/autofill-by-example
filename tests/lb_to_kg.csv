1,0.454545454545455
2,0.909090909090909
2,0.909090909090909
3,
10,
5,
4,
19,
2,
1,
9,
7,
4,
3,
12,
4,
1,
26,
8,
3,
3,
8,
4,