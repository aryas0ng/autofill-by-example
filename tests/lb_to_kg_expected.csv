1,0.454545454545455
2,0.909090909090909
2,0.909090909090909
3,1.36363636363636
10,4.54545454545455
5,2.27272727272727
4,1.81818181818182
19,8.63636363636364
2,0.909090909090909
1,0.454545454545455
9,4.09090909090909
7,3.18181818181818
4,1.81818181818182
3,1.36363636363636
12,5.45454545454545
4,1.81818181818182
1,0.454545454545455
26,11.8181818181818
8,3.63636363636364
3,1.36363636363636
3,1.36363636363636
8,3.63636363636364
4,1.81818181818182